src/cli/cbenv_bucket.rs

```diff
@@ -0,0 +1,64 @@
+use crate::cli::util::{no_active_cluster_error, NuValueMap};
+use crate::state::State;
+use std::sync::{Arc, Mutex};
+
+use nu_engine::CallExt;
+use nu_protocol::ast::Call;
+use nu_protocol::engine::{Command, EngineState, Stack};
+use nu_protocol::{Category, PipelineData, ShellError, Signature, SyntaxShape};
+
+#[derive(Clone)]
+pub struct UseBucket {
+    state: Arc<Mutex<State>>,
+}
+
+impl UseBucket {
+    pub fn new(state: Arc<Mutex<State>>) -> Self {
+        Self { state }
+    }
+}
+
+impl Command for UseBucket {
+    fn name(&self) -> &str {
+        "cb-env bucket"
```

Comment:
Open to better names than cb-env but given other nushell commands I think it makes sense.

Reply:
>  Are there popular extensions

No :)